### ISSUE TITLE
[Ubuntu][Noble] virtme-ng can't enter to shell

### ISSUE THREAD

**Issue by kobataiwan** [arighi/virtme-ng#152]
Symptom:  
On Noble, install virtme-ng=1.22-1,
then run virtme-ng
```
$sudo virtme-ng -r v6.9
```
vng is simply finished.

With virtme-ng=1.25+2-1ubuntu1, the issue isn't observed.

**Comment by arighi** [comment]
Hey Koba, about this issue, IIUC we just need to update virtme-ng in Noble right? In that case maybe I can ping someone to ask for a resync... Apparently we already have vng 1.27-1 in Oracular, so syncing that in Noble should solve this issue.

**Comment by kobataiwan** [comment]
@arighi hi, Yap, resync the noble's virtme-ng to latest will resolve this issue. thanks

**Comment by arighi** [comment]
Good point, I'll try to open a bug and ask to resync. Otherwise I'll see if I can backport a patch to fix the specific issue.

**Comment by helen-fornazier** [comment]
Hi, I was wondering if there is any update on this.

@arighi did you open a bug to re-sync?

Thanks everyone.

**Comment by helen-fornazier** [comment]
I just opened it here: https://bugs.launchpad.net/ubuntu/+source/virtme-ng/+bug/2104184

**Comment by arighi** [comment]
@helen-fornazier hm... I thought they were going to automatically sync with oracular, but apparently they didn't... I'll prepare a fix and add it to your bug. Thanks!
